src/styles/styles.css

```diff
@@ -27,12 +22,15 @@ body {
 body,
 .book,
 .book-summary {
-  font-family: "Gotham SSm A", "Gotham SSm B", "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-family: "Overpass", "Helvetica Neue", Helvetica, Arial, sans-serif;
 }
+
+/*****************
+TOP BAR
+**********************/
 #platform-bar {
-  background: #000;
+  background: #302F45;
   color: #fff;
```

Comment:
Can this also be a var reference, even if it's not one of the brand colors proper?